docs/admin_guide.md

```diff
@@ -1,22 +1,22 @@
 # Admin Bare Metal/VM Guide
 
 Below is a step-by-step guide of the process for creating your own /kbin instance from the moment a new VPS/VM is created or directly on bare-metal.  
-This is a preliminary outline that will help you launch an instance for your own needs.
+This is a preliminary outline that will help you launch an instance for your needs.
 
-For Docker see: [Admin Deployment Guide](./docker_deployment_guide.md).
+For Docker, see: [Admin Deployment Guide](./docker_deployment_guide.md).
 
 > **Note**
 > /kbin is still in the early stages of development.
 
 If you would like to support the project, you can register using the following [affiliate link](https://hetzner.cloud/?ref=8tSPCw0qqIwl).
 
-This guide is aimed for Debian / Ubuntu distribution servers, but it could run on any modern Linux distro. This guide will however uses the `apt` commands.
+This guide is aimed for Debian / Ubuntu distribution servers, but it could run on any modern Linux distro. This guide will, however, will use the `apt` commands.
```

Comment:
This is still bad; you repeated "will". Just make it "will, however, use"

Reply:
Hey, you can suggest changes on GitHub by specifying the triple-tick language to be "suggestion" and filling in your preferred version of the line like this!
```suggestion
This guide is aimed for Debian / Ubuntu distribution servers, but it could run on any modern Linux distro. This guide will, however, use the `apt` commands.
```
This way, they can easily implement your requests and you'll also get credited as the author of the commit.